From d74091a7452846112c0bb11e1412cd2af48ee7cd Mon Sep 17 00:00:00 2001
From: Carlo Buccisano <slashdeipoveri@gmail.com>
Date: Tue, 12 Dec 2023 00:24:20 +0100
Subject: [PATCH] The function file_valid_extension should return True if
 plugin is different then NULL, not if it is equal.

---
 zathura/utils.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/zathura/utils.c b/zathura/utils.c
index 802757f..8f379be 100644
--- a/zathura/utils.c
+++ b/zathura/utils.c
@@ -60,7 +60,7 @@ file_valid_extension(zathura_t* zathura, const char* path)
   zathura_plugin_t* plugin = zathura_plugin_manager_get_plugin(zathura->plugins.manager, content_type);
   g_free(content_type);
 
-  return plugin == NULL;
+  return plugin != NULL;
 }
 
 static void
-- 
2.43.0

